src/main.js

```diff
@@ -27,6 +29,18 @@ function createWindow() {
 
     currentInterval = setInterval(updateNotifications.bind(this, app, win), REFRESH_RATE);
 
+    // Add support for tray icon
+    win.on('minimize', function (event) {
```

Comment:
change to on('close') please

Reply:
You will have to delete line 51-56 though if we use close, you should remove the app.on('window-all-closed') handler